.env

```diff
@@ -1,9 +1,24 @@
-export VAULT_URL=
-export VAULT_TOKEN=
-export LS_CLIENTNAME=
-export LS_SERVERNAME=
-export SERVER_NAME=
-export SLACK_WEBHOOK=
-export SLACK_USERNAME=WhateverMan
+# Docker
```

Comment:
Could be good to add to this comment - "The latest version can be found here - https://github.com/docker/compose/releases "

Reply:
Good call! Added in 902bdae.